src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -128,14 +135,46 @@ def init_from_file(self) -> None:
             profile_jsonc = commentjson.load(fileobj)
```

Comment:
If we assign to `self.profile_jsonc` instead, we will have the entire config file stored in memory as a class variable.